src/languages/es.js

```diff
@@ -474,12 +474,14 @@ export default {
         gotIt: 'Ok, entendido',
     },
     addPayPalMePage: {
-        enterYourUsernameToGetPaidViaPayPal: 'Escribe tu nombre de usuario para que otros puedan pagarte a través de PayPal.',
+        enterYourUsernameToGetPaidViaPayPal: 'Recibe pagos via PayPal.',
```

Comment:
```suggestion
        enterYourUsernameToGetPaidViaPayPal: 'Recibe pagos vía PayPal.',
```
My bad, should be 'vía' here.